pkg/network/protocols/http/etw_interface_test.go

```diff
@@ -191,7 +191,12 @@ func TestEtwTransactions(t *testing.T) {
 
 	etw.StartReadingHttpFlows()
 
-	time.Sleep(time.Second)
+	/*
+	 * This is a bit kludgy, but we need to wait for the ETW provider to start.  Empirically, it
+	 * takes "some time" for the provider to start sending messages, which leads to some raciness
+	 * if we're looking for very specific messages.
+	 */
+	time.Sleep(10 * time.Second)
```

Comment:
I wonder if in future you can feed (e.g. via channel events to the test itself to have better ability to control and adapt to variable ramp up time.